openpype/plugins/publish/extract_thumbnail.py

```diff
@@ -98,8 +99,18 @@ def process(self, instance):
                 self.log.debug("Trying to convert with OIIO")
                 # If the input can read by OIIO then use OIIO method for
                 # conversion otherwise use ffmpeg
+                colorspace_data = repre["colorspaceData"]
+                source_colorspace = colorspace_data["colorspace"]
+                config_path = colorspace_data.get("config", {}).get("path")
+                display = colorspace_data["display"]
+                view = colorspace_data["view"]
```

Comment:
Yeah, good points. Have updated the description with hosts and families to test.

Will run some more tests in Maya with enabled/disabled settings.

Reply:
This needs to be optional, since there are no `colorspaceData` at representation if color management is disabled at host or global level. @tokejepsen are you able to do it or should we take push this PR farther?

Reply:
@jakubjezek001 sorry, I dont think i'll have time to continue this PR anytime soon.

Reply:
@jakubjezek001 will you be picking this up?

Reply:
yes ;)